packages/mobile-first-router/src/adapters/queryStringAdapter.ts

```diff
@@ -75,9 +76,29 @@ const getRoute = (initialRoute) => {
   ];
 };
 
+const getTab = (initialTab: number) => {
+  const queryString = getQueryString(
+    path(['document', 'location', 'search'], global)
+  );
+
+  if (!queryString.tab || queryString.tab === `${initialTab}`) {
+    return initialTab;
+  }
+
+  return parseInt(queryString.tab) || initialTab;
+};
+
+const setTab = (action) => {
+  try {
```

Comment:
generally try to avoid try catches when possible for performance reasons, since this appears to be our current pattern we can change it up later